### PR TITLE
MOTECH-2785: Fixed problem with not closing modal

### PR DESCRIPTION
**Pull request by MichalKus**
When backspace button is pressed, modal is closed and we go back to previous site

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
test this please

File: platform/mds/mds-web/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -2042,6 +2042,15 @@
...
         };
+
+         $scope.$on('$stateChangeStart', function(event, toState, toParams, fromState) {
+            if(fromState.name === 'mds.schemaEditor') {
+               var modal = angular.element('#advancedObjectSettingsModal');
+               modal.modal('hide');
+            }
+         });
+
+
```
Comment (jredlarski) [inline platform/mds/mds-web/src/main/resources/webapp/js/controllers.js]: @MichalKus Remove redundant empty line

**Comment by jredlarski** [comment]
test this please

**Pull request merged**